client/src/components/App.jsx

```diff
@@ -1,43 +1,50 @@
 import React from 'react';
 import $ from 'jquery';
-import { FaBeer } from 'react-icons/fa';
-import { FaHeart } from 'react-icons/fa';
+
+import Photo from './Photo.jsx';
+import Content from './Content.jsx';
+
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
-    this.handleShareButtonClick = this.handleShareButtonClick.bind(this);
+    this.state = {
+      photos: [],
+      showContent: false,
+    };
     this.handleViewPhotosButtonClick = this.handleViewPhotosButtonClick.bind(this);
   }
 
-  handleShareButtonClick() {
-
-  }
   handleViewPhotosButtonClick() {
+    this.setState({
+      photoButtonClick: !this.state.photoButtonClick
```

Comment:
It's best to add a trailing comma, even for the last key-value pair in an object. (See the AirBnB Style Guide, which our team is using.)